generic-templates/newsletter-signup-snippet/snippet.html

```diff
@@ -435,10 +435,18 @@
         var trackItemWidth = 420;
         var currentIndex = 0;
 
-        emailInput.addEventListener("focus", function() {
+        emailInput.addEventListener("click", function() {
```

Comment:
This just clears the error state.

Before, you could only submit it by clicking, which would blur the emailInput, and if the user clicked on it again, it would reset.

With enter as submit, if it's not a valid email, it's still focused, and thus click it doesn't clear it.

You can test this like so:

1. type in an invalid email.
2. hit enter
3. click the input to fix your email

Expected: Should clear error state, because I'm fixing it.
Actual: it is still in an error state.

You can try the same steps, but instead of hitting enter, press the next button, and it should then work as expected.